packages/boot/test/bootstrapTests/vaults-integration.test.ts

```diff
@@ -222,41 +206,44 @@ test('close vault', async t => {
   });
 });
 
-test('open vault with insufficient funds gives helpful error', async t => {
-  const { walletFactoryDriver } = t.context;
-
-  const wd = await walletFactoryDriver.provideSmartWallet(
-    'agoric1insufficient',
-  );
+test.serial(
+  'open vault with insufficient funds gives helpful error',
+  async t => {
+    const { walletFactoryDriver } = t.context;
```

Comment:
Formatting nit: it would be nice to avoid this indentation churn.
```suggestion
test.serial('excessive debt error contains helpful details', async t => {
  const { walletFactoryDriver } = t.context;
```